src/middleware.py

```diff
@@ -1,51 +1,22 @@
-from fastapi import Request, status
-from fastapi.responses import Response, JSONResponse
+import uuid
+from fastapi import Request
+from fastapi.responses import Response
 from starlette.middleware.base import BaseHTTPMiddleware
-import jwt
 
 
-SECRET_KEY = 'you will never guess'
-
-
-class sessionMiddleware(BaseHTTPMiddleware):
+class SessionMiddleware(BaseHTTPMiddleware):
     def __init__(self, app):
         super().__init__(app)
 
     async def dispatch(self, request: Request, call_next) -> Response:
 
         session = request.cookies.get('session', None)
 
-        if is_auth_endpoint(request):
-            if session:
-                return JSONResponse({'status': 'authorized'}, status.HTTP_202_ACCEPTED)
-            else:
-                return await call_next(request)
+        # TODO change request.state.player to change request.state.session
+        request.state.player = session
+        response = await call_next(request)
 
         if not session:
-            response = JSONResponse({'error': 'Unauthorized'}, status_code=status.HTTP_401_UNAUTHORIZED)
-            response.headers['WWW-Authentication'] = 'URI /auth'
-            return response
-        else:
-            request.state.player = _get_player_uuid(session)
-            response = await call_next(request)
+            response.set_cookie(key='session', value=str(uuid.uuid4()))
```

Comment:
I think we can start upgrading session structure from now:
```suggestion
            session = {
                'id': str(uuid.uuid4()),
            }
            response.set_cookie(key='session', value=session)
```

Reply:
Do you think, we need `id` key for any purposes?

Reply:
Well, we could use it as "player_id", but the main reason of my suggestion is that we need a session which is able to hold bigger data, not only player ID (game ID, metadata, or whatever). For example, session objects in Flask applications are always mapping objects.

Reply:
We decided to resolve this question later